src/components/dashboard/EditButton.tsx

```diff
@@ -0,0 +1,17 @@
+'use client'
+
+import React from 'react'
+import { Button } from '../ui/button'
+import { FaEdit } from 'react-icons/fa'
+
+export default function EditButton({ id }: { id: string }) {
+  return (
+    <Button
```

Comment:
Here since you want this to be a link styled like a button, I think you need the `asChild` attribute. Checkout shadcn's documents for more: https://ui.shadcn.com/docs/components/button#link

Reply:
Double check this but I added custom logic that handles converting this to a `Link` component using the `asLink` property in the source code for the `button.tsx`. Does that change what your concern is?

Reply:
> Double check this but I added custom logic that handles converting this to a `Link` component using the `asLink` property in the source code for the `button.tsx`. Does that change what your concern is?

It does, thanks! You're good 😎